src/content/self-hosted/install/releases.mdx

```diff
@@ -63,9 +63,9 @@ November 25, 2022
 
 - Update okteto cli version to [2.9.0](https://github.com/okteto/okteto/releases/tag/2.9.0)
 - Support for self-signed certificates and private CA with Docker
-- Buildkit port is now [configurable](/docs/self-hosted/administration/configuration/#buildkit)
+- Buildkit port is now [configurable](self-hosted/administration/configuration.mdx#buildkit)
```

Comment:
I've been testing these and the links in `releases.mdx` do not change the current version. I think this should be an exception since one could want to check out what was in the 1.2 version and this link should guide them to that version and not the one they are currently on.
Repro steps:
- select 1.4
- go to release notes (this file)
- select a link from the release notes of 1.2 (like this one)
- you'll be redirected to the 1.4 page (what I'm pointing out is that we should redirect to 1.2)

WDYT? cc @rberrelleza @pchico83

Reply:
Ah, that's an interesting point. At the same time, sending them to another version might be confusing if they are looking at the release notes.  Let me do some research about what other products do.  But let's not block this change due to that, we can iterate on a separate PR